parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -55,6 +57,9 @@
               CodecFactory.BytesCompressor.class,
               MessageType.class,
               ByteBufferAllocator.class,
+              int.class,
+              boolean.class,
+              InternalFileEncryptor.class,
```

Comment:
Looks like `ColumnChunkPageWriteStore` has been public since this commit: https://github.com/apache/parquet-mr/commit/89c5c25d44f8959d1a66493c06630ab0060cf346

That's in all 1.12.x releases, so I think we can remove the use of reflection.

Reply:
Oh, cool!!